handlers/main.yml

```diff
@@ -23,13 +23,20 @@
       name: "{{ rhel7stig_time_service }}"
       state: restarted
 
-- name: init aide status
-  async_status:
-    jid: "{{ rhel7stig_aide_init_status.ansible_job_id }}"
-  register: rhel7stig_aide_init_job_result
-  until: rhel7stig_aide_init_job_result.finished
-  retries: 60
-  delay: 5
+- name: init aide
+  command: /usr/sbin/aide --init -B 'database_out=file:{{ rhel7stig_aide_db_output_file }}'
```

Comment:
Probably want to move the aide init to after any other handlers that make filesystem changes.

Reply:
Hmm is this as simple as moving these handlers to the end of the file?

Reply:
handlers are always executed in the order specified in the handlers file, regardless of the order they were notified.

Reply:
We should probably move the reboot handler to the bottom of the file then.

Reply:
True, though I don't think we use the reboot handler.